data/real/basic.lean

```diff
@@ -176,6 +187,8 @@ let ⟨M, M0, H⟩ := f.bounded' 0 in
 ⟨M, mk_le_of_forall_le ⟨0, λ i _,
   rat.cast_le.2 $ le_of_lt (abs_lt.1 (H i)).2⟩⟩
 
+attribute [irreducible] real comm_ring_aux
```

Comment:
Could you put a bit of documentation here to make it more obvious and also explain why this is being done?